.gitignore

```diff
@@ -56,4 +56,4 @@ gdbinit
 src/components/lib/libc/musl-1.2.0/include/asm/
 src/components/lib/libc/musl-1.2.0/include/asm-generic/
 src/components/lib/libc/musl-1.2.0/include/linux/
-src/components/lib/libc/musl-1.2.0/include/sys/queue.h
\ No newline at end of file
+src/components/lib/libc/musl-1.2.0/include/sys/queue.h
```

Comment:
@betahxy Is this an error? Looks like this comes from a merge with your code. I might have done this incorrectly.

Reply:
@gparmer This is OK. I'll add this back. Not a big problem.